resolvers/webpack/README.md

```diff
@@ -4,6 +4,9 @@
 
 Webpack-literate module resolution plugin for [`eslint-plugin-import`](https://www.npmjs.com/package/eslint-plugin-import).
 
+> :warn: Only "synchronous" Webpack configs supported at that moment. If your config returns `Promise`, this will cause problems.
```

Comment:
```suggestion
> :warn: Only "synchronous" Webpack configs are supported at the moment. If your config returns a `Promise`, this will cause problems.
```